frontend/packages/console-app/src/components/network-policies/network-policy-form.tsx

```diff
@@ -166,6 +179,32 @@ export const NetworkPolicyForm: React.FunctionComponent<NetworkPolicyFormProps>
 
   return (
     <Form onSubmit={save} className="co-create-networkpolicy">
+      {showSDNAlert &&
+        networkFeaturesLoaded &&
+        networkFeatures?.PolicyEgress === undefined &&
+        networkFeatures?.PolicyPeerIPBlockExceptions === undefined && (
+          <Alert
+            variant="info"
+            title={<p>{t('public~When using the OpenShift SDN cluster network provider:')}</p>}
+            actionClose={<AlertActionCloseButton onClose={() => setShowSDNAlert(false)} />}
+          >
+            <ul>
+              <li>{t('public~Egress network policy is not supported.')}</li>
+              <li>
+                {t(
+                  'public~IP block exceptions are not supported and would cause the entire IP block section to be ignored.',
+                )}
+              </li>
+            </ul>
+            <p>
+              {t('public~More information:')}&nbsp;
```

Comment:
I would add another sentence to help the user understand what he has to do with this message, something like: "Refer to your cluster administrator to know which network provider is used".